### PR TITLE
Added Some Snippets

### PR DESCRIPTION
**Pull request by Tetrax-10**

### CODE REVIEW

**Review by CharlieS1103 — APPROVED** [review]
LGTM, I think as long as padster has no changes you're good to go.

**Comment by theRealPadster** [comment]
> LGTM, I think as long as padster has no changes you're good to go.

Lol you guys keep acting like I don't have a pending review with comments ..

**Review by theRealPadster — CHANGES_REQUESTED** [review]

**Comment by CharlieS1103** [comment]
> > LGTM, I think as long as padster has no changes you're good to go.
> 
> Lol you guys keep acting like I don't have a pending review with comments ..

Wait you did? For whatever reason I didn't see that. Unless of course that's a joke in which case I also didn't see that.

File: packages/marketplace/src/resources/snippets.ts [hunk 1]
```diff
@@ -107,10 +107,46 @@ export default [
...
+    "preview": "resources/assets/snippets/pointer.png",
+  },
+  {
+    "title": "Remove Top gradient",
+    "description": "Removes gradient from home page and playlist page",
+    "code": ".main-entityHeader-backgroundColor { display: none !important; } .main-actionBarBackground-background { display: none !important; } .main-home-homeHeader { display: none !important; }",
+    "preview": "resources/assets/snippets/remove-gradient.png",
```
[the remark this answers is not in the document]
Comment (Tetrax-10) [reply packages/marketplace/src/resources/snippets.ts:132]: it works like i just ported it from my theme

File: packages/marketplace/src/resources/snippets.ts [hunk 2]
```diff
@@ -107,10 +107,46 @@ export default [
...
+  },
+  {
+    "title": "Fix Liked Button",
```
[the remark this answers is not in the document]
Comment (Tetrax-10) [reply packages/marketplace/src/resources/snippets.ts:135]: when clicking the color will be green this one actually fixes it

[the remark this answers is not in the document]
Comment (theRealPadster) [reply packages/marketplace/src/resources/snippets.ts:135]: @kyrie25 Didn't you fix this with something in the go file or something? Or am I imagining things?

[the remark this answers is not in the document]
Comment (kyrie25) [reply packages/marketplace/src/resources/snippets.ts:135]: I don't remember doing so? The one I fixed was the button color on highlight and pressed.

[the remark this answers is not in the document]
Comment (Tetrax-10) [reply packages/marketplace/src/resources/snippets.ts:135]: yeah while clicking liked button that animation color will be fixed by this snippet

File: packages/marketplace/src/resources/snippets.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (CharlieS1103) [reply packages/marketplace/src/resources/snippets.ts:132]: It does work

File: packages/marketplace/src/resources/snippets.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (theRealPadster) [reply packages/marketplace/src/resources/snippets.ts:135]: Ahh good stuff 👍

**Review by theRealPadster — APPROVED** [review]

**Pull request merged**